docs/resources/risks/risk-score.md

```diff
@@ -133,29 +133,29 @@ How long the strategy has been running live on yearn.finance:
   </tr>
   <tr>
    <td>5</td>
-   <td>New code. Did not go to ape tax before going live on yearn.finance</td>
+   <td>Code is new and did not go to ape tax before going live on yearn.finance</td>
   </tr>
   <tr>
    <td>4</td>
-   <td>Code has been live less than a month</td>
+   <td>Code has been running for less than one month</td>
   </tr>
   <tr>
    <td>3</td>
-   <td>Code has been live 1-4 months</td>
+   <td>Code has been running between 1-4 months</td>
   </tr>
   <tr>
    <td>2</td>
-   <td>Code live 4+ months</td>
+   <td>Code has been running for 4+ months</td>
   </tr>
   <tr>
    <td>1</td>
-   <td>Code live 8+ months. No critical issues and no changes in code base over this time</td>
+   <td>Code has been running for 8+ months with no critical issues and no changes in code base</td>
   </tr>
 </table>
 
 ### Protocol Safety
 
-Internal evaluation from yearn’s point-of-view of how resilient are the protocols that the strategy works with. It regards safety measures given the current DeFi security standards, it's heavily based on informed opinions regarding our internal assessments and due diligence compared to the top projects in DeFI. Accounts for multisig health, decentralization, bounty programs, audits, etc. 
+Internal evaluation from yearn’s point-of-view of how resilient are the protocols that the strategy works with. It takes into account safety measures given the current DeFi security standards, based on our internal assessments and due diligence compared to the top projects in DeFI. This includes multisig health, decentralization, bounty programs, audits, etc. 
```

Comment:
Protocol Safety evaluates the resilience of the protocol the strategy uses. It takes into account...

Reply:
```suggestion
Protocol Safety evaluates the resilience of the protocol the strategy uses. It takes into account how resilient are the protocols that the strategy works with. It takes into account safety measures given the current DeFi security standards, based on our internal assessments and due diligence compared to the top projects in DeFI. This includes multisig health, decentralization, bounty programs, audits, etc. 
```

Reply:
```suggestion
Protocol Safety evaluates the resilience of the protocol the strategy uses. It takes into account the safety measures given the current DeFi security standards, based on our internal assessments and due diligence compared to the top projects in DeFI. This includes multisig health, decentralization, bounty programs, audits, etc. 
```